rubygem-smart_proxy_dynflow/rubygem-smart_proxy_dynflow.spec

```diff
@@ -54,6 +54,9 @@ This package contains documentation for rubygem-%{gem_name}.
 %install
 mkdir -p %{buildroot}%{gem_dir}
 
+mkdir -p %{buildroot}%{_localstatedir}/lib/foreman-proxy/dynflow
+chown foreman-proxy:foreman-proxy %{buildroot}%{_localstatedir}/lib/foreman-proxy/dynflow
```

Comment:
In a similar train of thought, it'd be useful I think to specify a restrictive mode for the directory (e.g. 0700 or 0750) to prevent any user reading the DB.